### DISCUSSION TITLE
Delete on Restart but not on Tab Close?

### DISCUSSION THREAD

**Discussion by PeterL-77** [Cookie-AutoDelete/Cookie-AutoDelete#1597] [Q&A / Support] [answered]
Is it possible to delete cookies on browser restart but not on tab close? I either get both or none, can't figure out how to do only one.

**Comment by kennethtran93** [comment, marked as answer]
https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/wiki/FAQ:-Common-Questions-and-Issues#how-do-i-set-all-cookies-to-be-deleted-on-browser-restart-rather-than-on-tab-close

**Comment by PeterL-77** [comment]
Thank you, bro. I looked briefly at FAQs but apparently too briefly.
